sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -140,36 +177,42 @@ def _to_internal_entity(self):
         self._internal_qd.requires_session = self.requires_session
         self._internal_qd.status = self.status
         self._internal_qd.support_ordering = self.support_ordering
-        self._internal_qd.created_at = self.created_at
+        self._internal_qd.forward_to = self.forward_to
+        self._internal_qd.forward_dead_lettered_messages_to = self.forward_dead_lettered_messages_to
+        self._internal_qd.user_metadata = self.user_metadata
 
         return self._internal_qd
 
 
 class QueueRuntimeInfo(object):
     """Service Bus queue metrics.
 
-    :param queue_name: Name of the queue.
-    :type queue_name: str
-    :param accessed_at: Last time a message was sent, or the last time there was a receive request
-     to this queue.
-    :type accessed_at: ~datetime.datetime
-    :param created_at: The exact time the queue was created.
-    :type created_at: ~datetime.datetime
-    :param updated_at: The exact time a message was updated in the queue.
-    :type updated_at: ~datetime.datetime
-    :param size_in_bytes: The size of the queue, in bytes.
-    :type size_in_bytes: int
-    :param message_count: The number of messages in the queue.
-    :type message_count: int
-    :param message_count_details: Details about the message counts in queue.
-    :type message_count_details: ~azure.servicebus.management.MessageCountDetails
     """
 
     def __init__(
         self,
         **kwargs
     ):
-        self.queue_name = kwargs.get('queue_name', None)
+        """
+        :keyword name: Name of the queue.
+        :type name: str
+        :keyword accessed_at: Last time a message was sent, or the last time there was a receive request
+         to this queue.
+        :type accessed_at: ~datetime.datetime
+        :keyword created_at: The exact time the queue was created.
+        :type created_at: ~datetime.datetime
+        :keyword updated_at: The exact time a message was updated in the queue.
+        :type updated_at: ~datetime.datetime
```

Comment:
These should be documented as :ivar: - as we never expect a user to instantiate this object.